cyvlfeat/_vl/gmm.pxd

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Author: Alexis Mignon <alexis.mignon@probayes.com>
+# Date: Wed Jun  1 14:58:43 2016
+# Copyright (c) 2016 ProbaYes SAS
```

Comment:
I'm not sure I want to start maintaining copyright notices at the top of each file. Would you be happy if we added an AUTHORS.md in the root that listed all contributors/joint copyright holders. Thus ensuring your contributions fall under the BSD license as well?